torchgeo/datasets/vhr10.py

```diff
@@ -159,10 +159,7 @@ class VHR10(NonGeoDataset):
         "md5": "d30a7ff99d92123ebb0b3a14d9102081",
     }
     target_meta = {
-        "url": (
-            "https://raw.githubusercontent.com/chaozhong2010/VHR-10_dataset_coco/"
-            "master/NWPU%20VHR-10_dataset_coco/annotations.json"
-        ),
+        "url": "https://raw.githubusercontent.com/chaozhong2010/VHR-10_dataset_coco/ce0ba0f5f6a0737031f1cbe05e785ddd5ef05bd7/NWPU%20VHR-10_dataset_coco/annotations.json",  # noqa: E501
```

Comment:
We should never download files from the master branch, they may change or move to a different location. Instead, we should use permalinks when available.